src/compressed_tensors/utils/match.py

```diff
@@ -46,23 +49,70 @@ def match_named_modules(
     :param targets: target strings, potentially containing "re:" prefixes
     :param ignore: targets to ignore, potentially containing "re:" prefixes
     :param warn_on_fail: if True, warns if any targets do not match any modules in model
+    :param warn_on_unmatched_ignores: if True, warns if any ignores do not match any modules in model
+    :param yield_matched_targets: if True, yields the matched targets in addition to the module name and module
+    :param preprocess_name: a function to preprocess the module name
```

Comment:
Again, it'd be nice to do any custom changes outside of this bare-bones utils function

Reply:
Ahh okay, I didn't realize we'd removed that.